### PR TITLE
Added fix for #issue-#506

### PR DESCRIPTION
**Pull request by jtj9817**
Added a Promises-based test to check if the data was successfully written and that the file is not empty.

### CODE REVIEW

**Comment by jrkong** [comment]
The changes to package-lock.json don't look like they are related to your change. If it isn't important to your PR can you remove them?

**Comment by JoshuaRM** [comment]
Based on the specifications you included in issue #506 it looks like you nailed it! The code you provided full supports promises, and does not affect the execution of the function. I ran tests on your code to make sure everything ran correctly, and I'm pleased to say no errors were returned. All tests ran successfully and passed the tester on the first attempt.

File: tests/spec/fs.write.spec.js [hunk 1]
```diff
@@ -59,4 +59,22 @@ describe('fs.write', function() {
       });
     });
   });
+
+});
+
+describe('fsPromises.writeFile', function () {
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+  it('should write data to a file', () => {
```
Comment (humphd) [inline tests/spec/fs.write.spec.js]: Add a newline above this please.

File: tests/spec/fs.write.spec.js [hunk 2]
```diff
@@ -59,4 +59,22 @@ describe('fs.write', function() {
...
+    var fs = util.fs().promises;
+    var buffer = new Filer.Buffer([1, 2, 3, 4, 5, 6, 7, 8]);
+
+
```
Comment (humphd) [inline tests/spec/fs.write.spec.js]: Remove the extra newline here

**Comment by humphd** [comment]
This test has already found it's way into master via another student's PR.  Thanks for starting it, I'm going to close.

**Pull request closed without merging**